cypress/e2e/with_mock_data/items.cy.ts

```diff
@@ -930,12 +931,12 @@ describe('Items', () => {
         expect(patchRequests.length).equal(1);
         const request = patchRequests[0];
         expect(JSON.stringify(await request.json())).equal(
-          '{"file_name":"test file","description":"test description","title":"test title"}'
+          '{"file_name":"test file.png","description":"test description","title":"test title"}'
         );
       });
     });
 
-    it('not changing any fields shows error', () => {
+    it('shows error when no fields have been changed', () => {
```

Comment:
```suggestion
    it('shows error message when no fields have been changed', () => {
```
apologies i know i changed it to this the first time :)